src/burn/engine/variable.cpp

```diff
@@ -240,7 +242,7 @@ extern "C" HRESULT VariableInitialize(
         {L"ProgramFilesFolder", InitializeVariableCsidlFolder, CSIDL_PROGRAM_FILESX86},
 #else
         {L"ProgramFiles64Folder", InitializeVariableRegistryFolder, CSIDL_PROGRAM_FILES},
-        {L"ProgramFilesFolder", InitializeVariableCsidlFolder, CSIDL_PROGRAM_FILES},
+        {L"ProgramFilesFolder", InitializeVariableCsidlFolder, CSIDL_PROGRAM_FILES },
```

Comment:
Not a big deal, but cleaner if you get rid of the now-added space. Probably just if you have any other changes to make.